mods/dualwielding/scripts.js

```diff
@@ -110,10 +110,10 @@ exports.BattleScripts = {
 			}
 			return false;
 		},
-		useItem() {
+		useItem(unused, source) {
```

Comment:
Why is the first param `itemName` unused?

Reply:
Because it's unused. Only Red Card actually passes any arguments to `useItem`, and it always passes `null` for the first argument. (`BattlePokemon.useItem` actually declares three arguments, but nobody ever uses the third. Did you want a PR to remove the unused arguments?)

Reply:
Hm, maybe later.